.github/workflows/macos-compile.yml

```diff
@@ -0,0 +1,29 @@
+# Platform specific MacOS test. See verible-ci for main continuous integration.
+name: macos-compile
+
+on:
+  push:
+    branches:
+    - master
+    - main
+  pull_request:
+    branches:
+    - master
+    - main
+
+jobs:
+  MacOsBuild:
+    runs-on: macos-latest
```

Comment:
comment on what version OS this happens to be?  other known versions of OS X Developer Tools?

Reply:
Version is unknown. Whatever github choses to be 'latest'.
Given that this is just meant as a litmus test, we actually don't really want to know, we just want to make sure that whatever typical latest macos there is, it will compile.